util/src/test/scala/org/alephium/util/ValueSortedMapSpec.scala

```diff
@@ -51,4 +51,15 @@ class ValueSortedMapSpec extends AlephiumSpec {
     map.min is 'a'
     map.max is 'b'
   }
+
+  it should "remove elements" in {
+    import akka.util.ByteString
+    import Bytes.byteStringOrdering
+    val map    = ValueSortedMap.empty[ByteString, TimeStamp]
+    val hashes = (0 until 100).map(_ => UnsecureRandom.nextU256().toBytes)
+    val now    = TimeStamp.now()
+    hashes.foreach(map.put(_, now))
+    map.remove(hashes.head)
+    map.remove(hashes.head)
```

Comment:
I don't see any assertion, but I suppose previously removing twice the key was throwing an error

Reply:
Yes, the double removal does not work previously